settings.ini

```diff
@@ -25,7 +25,9 @@ license = apache2
 status = 2
 
 # Optional. Same format as setuptools requirements
-requirements = fastcore torch torchvision
+requirements = fastcore tqdm matplotlib torch torchvision
```

Comment:
Can we take out torch and torchvision from here and specify it should be previously installed?

Reply:
We can take that out. But we need to ensure that the user has these two installed. If they aren't then how would we handle?

Reply:
While testing Torch triggered a wrong package, pytorch should be done by the user. We will automate fastai and lightning. Agreed and changed. Torch and torhvision go hand in hand so better user handle from PyTorch.